### PR TITLE
Update Atlas Help Content

### PR DESCRIPTION
**Pull request by bensojona**
Updating Atlas help content to reflect what the current Atlas UI looks like as well as some copy updates.

This must be deployed with the [Atlas images PR](https://github.com/hashicorp/atlas/pull/876) or some images may not show up.

Also need to merge https://github.com/hashicorp/atlas-examples/pull/42

### CODE REVIEW

File: getting-started/deploy-fully-configured-infrastructure.html.md [hunk 1]
```diff
@@ -14,17 +14,17 @@ Now that we have an artifact stored in Atlas, we can reference it in the Terrafo
 	}
 
 	provider "aws" {
-	  access_key = "ACCESS_KEY_HERE"
-	  secret_key = "SECRET_KEY_HERE"
+		access_key = "ACCESS_KEY_HERE"
```
Comment (KFishner) [inline getting-started/deploy-fully-configured-infrastructure.html.md]: hm this formatting looks weird

File: getting-started/deploy-fully-configured-infrastructure.html.md [hunk 2]
```diff
@@ -55,11 +84,11 @@ Now, let's combine the artifact with the [Terraform configuration written in the
 	    }
 
 	    health_check {
-	      healthy_threshold = 2
-	      unhealthy_threshold = 2
-	      timeout = 5
-	      target = "TCP:80"
-	      interval = 10
+			healthy_threshold = 2
```
Comment (KFishner) [inline getting-started/deploy-fully-configured-infrastructure.html.md]: formatting here looks odd too. i think we do 2 spaces everywhere

File: getting-started/deploy-fully-configured-infrastructure.html.md (hunk 1 above)
Comment (bensojona) [inline getting-started/deploy-fully-configured-infrastructure.html.md]: Yeah, saw that too. Here's what it looks like locally. 

image

File: getting-started/layout-infrastructure.html.md [hunk 3]
```diff
@@ -11,42 +11,72 @@ A Terraform configuration is a holistic view of your infrastructure — the desi
...
+		}
 	}
 
-
 	resource "aws_instance" "web" {
-	  	instance_type = "t1.micro"
-	  	ami = "ami-408c7f28"
+	    instance_type = "t1.micro"
+		ami = "ami-408c7f28"
+	    security_groups = ["${aws_security_group.allow_all.name}"]
```
Comment (KFishner) [inline getting-started/layout-infrastructure.html.md]: formatting

File: getting-started/layout-infrastructure.html.md [hunk 4]
```diff
@@ -114,13 +144,19 @@ terminal session and checking that `terraform` is available. By executing
...
 
 	Available commands are:
-	    apply      Builds or changes infrastructure
-	    graph      Create a visual graph of Terraform resources
-	    output     Read an output from a state file
-	    plan       Generate and show an execution plan
-	    refresh    Update local state file against real resources
-	    show       Inspect Terraform state or plan
-	    version    Prints the Terraform version
+		apply      Builds or changes infrastructure
```
Comment (KFishner) [inline getting-started/layout-infrastructure.html.md]: formatting

File: getting-started/layout-infrastructure.html.md [hunk 5]
```diff
@@ -133,42 +169,51 @@ Otherwise, Terraform is installed and ready to go!
...
+
+The log output will be similar to what is copied below. We've truncated some of the output to save space.
 
 	...
 
 	+ aws_elb.web
-	    availability_zones.#:          "" => "<computed>"
-	    dns_name:                      "" => "<computed>"
+		availability_zones.#: "" => "<computed>"
+		connection_draining:  "" => "0"
```
Comment (KFishner) [inline getting-started/layout-infrastructure.html.md]: formatting

File: getting-started/deploy-fully-configured-infrastructure.html.md (hunk 2 above)
Comment (bensojona) [inline getting-started/deploy-fully-configured-infrastructure.html.md]: Ah, I did 4 spaces :/ I'll update

File: getting-started/layout-infrastructure.html.md [hunk 6]
```diff
@@ -194,18 +239,40 @@ create real resources. You can execute changes by clicking "Confirm & Apply"
...
-	Apply complete! Resources: 2 added, 0 changed, 0 destroyed.
+	Apply complete! Resources: 4 added, 0 changed, 0 destroyed.
 
 	...
 
 Done! You can go to the AWS console to prove to yourself that the
 EC2 instances have been created.
 
-You can view the history of infrastructure changes in the environments tab of your [Atlas dashboard](http://atlas.hashicorp.com/environments). Click the environment name, and then "Changes" in the left navigation. Below is an example screenshot:
+You can also view the history of infrastructure changes in the "Changes" tab of your [Atlas environment](http://atlas.hashicorp.com/environments). Below is an example screenshot:
```
Comment (KFishner) [inline getting-started/layout-infrastructure.html.md]: "also" can probably be removed

**Comment by KFishner** [comment]
content updates are great! I'd just check the formatting to make sure it's 2 spaces errwhere.

**Comment by bensojona** [comment]
Will do!

**Comment by bensojona** [comment]
@KFishner so this is tab formatting vs space formatting, should I just convert it all to spaces?

**Comment by KFishner** [comment]
@bensojona yeah let's start being consistent around 2 spaces

**Comment by bensojona** [comment]
@KFishner Errright, everything has 2 spaces now

**Pull request merged**